tests/e2e/Adapter/Scopes/DocumentTests.php

```
@@ -7722,4 +7722,122 @@ public function testRegexInjection(): void
+                        Permission::read(Role::any()),
+                        Permission::create(Role::any()),
+                    ],
+                ]),
+            ]);
+            $this->fail('Expected DuplicateException');
+        } catch (DuplicateException $e) {
+            $this->assertNotEmpty($e->getMessage());
+        }
+
+        // With ignore, duplicates should be silently skipped
+        $count = $database->createDocuments(__FUNCTION__, [
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$count'. (undefined)

(UnusedLocalVariable)